README.md

```diff
@@ -14,11 +14,26 @@ This plugin adds a new `$clipboard` magic property to all of your Alpine compone
 
 ## Installation
 
+### NPM
+
+```bash
+npm install @ryangjchandler/alpine-clipboard
+```
+
+> Also available via yarn add @ryangjchandler/alpine-clipboard
```

Comment:
I don't think we need this note. Most people using `Yarn` will know that it can install all NPM package anyway.

Reply:
Removed. 👍